ArduPlane/adsb.cpp

```diff
@@ -98,14 +98,14 @@ void Plane::adsb_evasion_stop(void)
 {
     gcs_send_text(MAV_SEVERITY_CRITICAL, "ADS-B threat gone, continuing mission");
 
-    FlightMode prev_control_mode = control_mode;
-    set_mode(AUTO);
-    if (prev_control_mode == LOITER)
-    {
-        // if resuming from loiter, smoothly get back on track
-        prev_WP_loc = adsb_state.prev_wp;
-        auto_state.no_crosstrack = false;
-    }
+//    FlightMode prev_control_mode = control_mode;

```

Comment:
This was a total mistake! Must have checked in the wrong adsb.cpp. I had no intent of changing Plane. 


----- Original Message -----

From: "WickedShell" <notifications@github.com> 
To: "ArduPilot/ardupilot" <ardupilot@noreply.github.com> 
Cc: "Craig McDermott" <craig.mcdermott@mchsi.com>, "Author" <author@noreply.github.com> 
Sent: Monday, April 25, 2016 10:43:27 PM 
Subject: Re: [ArduPilot/ardupilot] ADSB sense and avoid experiment for Copter (#3984) 



In ArduPlane/adsb.cpp : 
> @@ -98,14 +98,14 @@ void Plane::adsb_evasion_stop(void)
>  {
>      gcs_send_text(MAV_SEVERITY_CRITICAL, "ADS-B threat gone, continuing mission");
>  
> -    FlightMode prev_control_mode = control_mode;
> -    set_mode(AUTO);
> -    if (prev_control_mode == LOITER)
> -    {
> -        // if resuming from loiter, smoothly get back on track
> -        prev_WP_loc = adsb_state.prev_wp;
> -        auto_state.no_crosstrack = false;
> -    }
> +//    FlightMode prev_control_mode = control_mode; 


Why disable planes end evasion code? I thought this was a copter PR not a plane one? 

— 
You are receiving this because you authored the thread. 
Reply to this email directly or view it on GitHub